packages/vscode-extension/src/utilities/launchConfiguration.ts

```diff
@@ -8,6 +8,13 @@ export function getLaunchConfiguration() {
   return ideConfig;
 }
 
+export function getLaunchConfigurations(): Array<LaunchConfigurationOptions> {
```

Comment:
do we still need the above method `getLauchConfiguration` ?? Would be better to remove it as someone may use it assuming there's only one configuration